Source/SvgAttributeCollection.cs

```diff
@@ -39,32 +39,48 @@ public TAttributeType GetAttribute<TAttributeType>(string attributeName, TAttrib
         /// </summary>
         /// <typeparam name="TAttributeType">The type of the attribute value.</typeparam>
         /// <param name="attributeName">A <see cref="string"/> containing the name of the attribute.</param>
+        /// <param name="inherited">The attribute is inherited or not.</param>
         /// <param name="defaultValue">The value to return if a value hasn't already been specified.</param>
         /// <returns>The attribute value if available; otherwise the ancestors value for the same attribute; otherwise the default value of <typeparamref name="TAttributeType"/>.</returns>

```

Comment:
Also maybe expand this a bit, as this is not self-explanatary, e.g. something like:
The attribute value if available and not set to "inherit"; the ancestors value for the same attribute if it exists 
and if either the attribute value is set to "inherit", or <paramref name="inherited"/> is true; the default value otherwise.

Reply:
Sorry to be picky, but the description you added for `inherited` actually was meant for the return value - sorry I wasn't more clear about that.
The description for `inherited` would be something like:
```Used only if the attribute value is not available. If set to true, the inherited value is returned in this case, otherwise the default value.```

Reply:
...and the comment from above belongs to `params`:

```The attribute value if available and not set to "inherit"; the ancestors value for the same attribute if it exists and if either the attribute value is set to "inherit", or <paramref name="inherited"/> is true; the default value otherwise.```

I will be more specific with documentation comments in the future to avoid these misunderstandings...